GhidraJupyterKotlin/src/main/java/GhidraJupyterKotlin/JupyterKotlinPlugin.java

```diff
@@ -39,31 +42,45 @@
 )
 //@formatter:on
 public class JupyterKotlinPlugin extends ProgramPlugin {
+	private static final String OPTION_LAST_URI = "LAST_URI";
+	private static final String DEFAULT_URI = "http://localhost:8888/tree";
```

Comment:
Maybe move to a properties file? Though there's probably no real justification...

Reply:
I saw that pattern of having the options key as a `static final String OPTION_` in the class elsewhere in the Ghidra code base when checking how to use options originally.
If you run `rg "static final String OPTION"` on a Ghidra checkout you get around 200 hits. Having the default value also there is something we mostly need because this can be configured on the tool or the program level